react/components/AutocompleteInput/index.tsx

```diff
@@ -253,7 +253,13 @@ const AutocompleteInput: React.FunctionComponent<AutocompleteInputProps> = ({
         roundedBottom={!popoverOpened}
         onKeyDown={handleKeyDown}
         onFocus={() => setShowPopover(true)}
-        onSearch={onSearch && (() => onSearch(term))}
+        onSearch={
```

Comment:
I think now it might be worth extracting to a `handleSearch` function. What do you think?

Reply:
Good idea! I just changed it, could you check?